src/shapes/Group.ts

```diff
@@ -582,6 +582,9 @@ export class Group
       targets: this.getObjects(),
       target: this,
     });
+    // dispose additional listeners that may have been
+    // added by the developer on the layoutManager #9787
+    this.layoutManager.dispose();
```

Comment:
My only thought about this is that the layout manager isn't bound to a single group so if an app uses a single layout manager for multiple groups this will break it so this PR forces a unique layout manager for each group but that is not strictly necessary, I designed it to be decoupled.

Reply:
I never knew about sharing LayoutManager. How would one dispose it then?

Reply:
If you mean to leave disposing to the app-level, then `dispose` in LayoutManager should be `noop` and have a comment explaining that the developer should do it accordingly to their needs, e.g. whether to dispose it on group disposal or canvas disposal.

Reply:
> My only thought about this is that the layout manager isn't bound to a single group so if an app uses a single layout manager for multiple groups this will break it so this PR forces a unique layout manager for each group but that is not strictly necessary, I designed it to be decoupled.

The sharing part was one of the contested parts of the design.
The group didn't have the layout manager, it had a bunch of methods to do the work of the layoutManager.
I asked if we could put those methods in a scoped box because with all the different ideas of layouts and functionality they were growing out of hand, and it was supposed to be a way to reduce clutter in the group. Period. The sharing between group is something that was added while you implemented the layoutManager but the usefulness of being shared isn't really clarified. It adds a lot of constrain to how this code should behave but what does it adds?
Indeed one of the first changes was to make every group generate its own layoutManager.

Can you make a case for the sharing part?

Reply:
This is the same as controls.
I did it like this because I know that is important for you.

```
class Group {
  static ownDefaults = {
    layoutManager: new LayoutManager() // shared across all instances
  }
}
```

Reply:
> Indeed one of the first changes was to make every group generate its own layoutManager.

I surfaced this when that happened IIRC

Reply:
It is not important for me, and i think it adds complexity both to signatures and usage.
The layout manager is just the group logic tucked away in a class. I don't see any need for sharing it across instances.